fastestimator/trace/metric/confusion_matrix.py

```diff
@@ -20,7 +20,7 @@
 
 from fastestimator.backend.to_number import to_number
 from fastestimator.trace.trace import Trace
-from fastestimator.util import Data
+from fastestimator.util.data import Data
```

Comment:
inconsistent with fact that Data was put into util init file

Reply:
done